packages/core/src/routes/admin-user/mfa-verifications.ts

```diff
@@ -82,7 +82,7 @@ export default function adminUserMfaVerificationsRoutes<T extends AuthedRouter>(
         const secret = generateTotpSecret();
         const service = ctx.URL.hostname;
         const user = getUserDisplayName({ username, primaryEmail, primaryPhone, name });
-        const keyUri = authenticator.keyuri(user, service, secret);
+        const keyUri = authenticator.keyuri(user ?? 'Unnamed User', service, secret);
```

Comment:
Can we put `Unnamed User` fallback in the `getUserDisplayName` util method, and use i18n instead of plain English?

Reply:
The fallback logic may differ, and the i18n should not be handled in "shared" package. Currently, Console already supports i18n for the default value, we can support it in "core" later.

Reply:
ok, but it seems all the fallbacks you are using in this pr are `?? 'Unnamed User'`. Any exceptions?

Reply:
Take a look at the "console" package